Ginger/Ginger/Actions/ActCreatePDFChartEditPage.xaml.cs

```diff
@@ -65,31 +73,38 @@ private void BrowseDataButton_Click(object sender, RoutedEventArgs e)
                     FileName = FileName.Replace(SolutionFolder, @"~\");
                 }
 
-                DataFileNameTextBox.ValueTextBox.Text = FileName;
-            }
+                DataFileNameTextBox.ValueTextBox.Text = FileName;                
+            }           
         }
 
-        private List<string> SetParamsCombo()
+        private void SetParamsCombo()
         {
+            List<string> tmp=new List<string>();
             object tempFileName = mAct.GetDataFileName();
-            if (ReferenceEquals(tempFileName,null)) return new List<string>();
```

Comment:
mAct.ParamList will retain the old value in case TempFileName is null. So better keep the return type as List<string> as before. and change in the call as per comment in text change event.